src/user/user.py

```diff
@@ -11,8 +17,7 @@ class User(ndb.Model):
     email = ndb.StringProperty(required=True)
     experience = ndb.IntegerProperty(default=0)
     name = ndb.StringProperty(required=True)
-    total_points = ndb.ComputedProperty(
-        lambda self: self._compute_total_points())
+    rank_data = ndb.StructuredProperty(Rating)
```

Comment:
`rank`?
`ranking`?
`rating`?

I'd like to avoid using `data` in our variable names (per clean code suggestion)